apps/condo/domains/common/components/HeaderActions.tsx

```diff
@@ -138,3 +142,20 @@ export const ButtonHeaderActions: React.FC<IRightButtonHeaderActionsProps> = (pr
 
     )
 }
+
+export const TabsAuthAction: React.FC<ITabsActionsProps> = (props) => {
+    const { currentActiveKey } = props
+    const intl = useIntl()
+    const registerTab = intl.formatMessage({ id: 'pages.auth.RegistrationTitle' })
+    const signInTab = intl.formatMessage({ id: 'SignIn' })
+    return (
+        <Tabs
+            defaultActiveKey={currentActiveKey}
+            style={ { alignItems:'center', fontWeight: 500 } }
```

Comment:
Move style to constant plz

Reply:
to move in constants styles with one style?